Makefile

```diff
@@ -228,8 +230,15 @@ test-integration: clean-cov generate fmt vet ginkgo ## Run Integration tests.
 		--coverpkg $(INTEGRATION_COVER_PKGS) \
 		--output-dir $(PROJECT_PATH)/coverage/integration \
 		--coverprofile cover.out \
-		--fail-fast \
```

Comment:
why remove `--fail-fast` ? is it incompatible with concurrent tests?

Reply:
It's mainly to align with Kuadrant Operator where the `fail-fast` flag was removed and by the recommendation at https://onsi.github.io/ginkgo/#recommended-continuous-integration-configuration.

I assume the flag works with concurrent tests. I don't feel too strongly with this so I can revert to keeping the flag